src/styles/variables.ts

```diff
@@ -89,7 +89,7 @@ export default {
     optionRowHeight: 64,
     optionRowHeightCompact: 52,
     optionsListSectionHeaderHeight: getValueUsingPixelRatio(32, 38),
-    overlayOpacity: 0.6,
+    overlayOpacity: 0.72,
```

Comment:
It's a little unclear if we want to change the overlay opacity app wide (variables.overlayOpacity) or just for the RHP (CONST.RIGHT_MODAL_BACKGROUND_OVERLAY_OPACITY)

cc @shawnborton or @dannymcclain may help here

Reply:
We want to change this for every single overlay/modal. cc @Expensify/design